rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/integration/ComponentTest.java

```diff
@@ -401,4 +417,528 @@ public void should_merge_component_with_failure_handling() throws IOException {
 
         assertEquals(HttpStatus.BAD_REQUEST, response.getStatusCode());
     }
+
+    @Test
+    public void should_create_component() throws Exception {
+        Map<String, String> newComponent = new HashMap<>();
+        newComponent.put("name", "Spring Framework");
+        newComponent.put("description", "The Spring Framework provides a comprehensive programming and configuration model for modern Java-based enterprise applications.");
+        newComponent.put("componentType", "OSS");
+        newComponent.put("homepage", "https://spring.io");
+
+        Component createdComponent = new Component();
+        createdComponent.setId("created-component-id");
+        createdComponent.setName("Spring Framework");
+        createdComponent.setDescription("The Spring Framework provides a comprehensive programming and configuration model for modern Java-based enterprise applications.");
+        createdComponent.setCreatedBy("admin@sw360.org");
+        createdComponent.setCreatedOn("2024-01-01");
```

Comment:
Please try to reuse existing component object
https://github.com/eclipse-sw360/sw360/blob/0ddfe73d6b4fb3290211c68dac155a5c8bfe7754/rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/integration/ComponentTest.java#L93-L99